draft-ietf-tls-deprecate-obsolete-kex.md

```diff
@@ -259,40 +251,21 @@ Given these problems, this document updates {{!RFC4346}}, {{!RFC5246}},
 
 # Non-Ephemeral Diffie Hellman {#non-ephemeral}
 
-Clients MUST NOT offer non-ephemeral FFDH cipher suites in TLS 1.2 connections.
+Clients and servers MUST NOT offer non-ephemeral FFDH cipher suites in TLS 1.2 connections.
```

Comment:
Nit: If I'm not mistaken, the server doesn't offer cipher suites, per se, but selects one. (From the specification of the ServerHello in RFC 5246: "The single cipher suite selected by the server from the list in ClientHello.cipher_suites.") Thus, I suggest: "Clients MUST NOT offer and servers MUST NOT select non-ephemeral..."

Reply:
Done, thanks :-)